templates/_entries/search/results.twig

```diff
@@ -0,0 +1,60 @@
+{#
+# Search Results Page
+# This will return all the entries with the client keyword.
+# excludeSearchEngine will exclude any page that has Exclude Search checked
+#}
+
+{# Import the Macro for search #}
+{% import "_macros/utils" as searchMacro %}
```

Comment:
Extract the search macros in its own file `_macros/search` because they seem tightly coupled to that specific function or rename to as `utils` instead of `searchMacro` as there are more than just search macros in this file.

Reply:
Rename to utils